vms/platformvm/create_chain_tx.go

```diff
@@ -63,13 +63,13 @@ func (tx *UnsignedCreateChainTx) Verify(
 		return errNilTx
 	case tx.syntacticallyVerified: // already passed syntactic verification
 		return nil
-	case tx.SubnetID.IsZero():
+	case tx.SubnetID == ids.Empty:
```

Comment:
This is broken, no?

Reply:
I don't think this is broken but it's also not necessary since it's logically equivalent to the next case. I'll remove